app/models/manageiq/providers/azure/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -1,4 +1,16 @@
 module ManageIQ::Providers::Azure::CloudManager::VmOrTemplateShared::Scanning
+  extend ActiveSupport::Concern
+
+  included do
+    supports :smartstate_analysis do
```

Comment:
as per our last meeting its ok to leave this here, because every provider has to implement ssa before we can mark it as supported